docker/observability/grafana/provisioning/dashboards/validator-dashboard.json

```diff
@@ -1760,7 +1760,7 @@
           },
           "editorMode": "code",
           "exemplar": true,
-          "expr": "histogram_quantile(0.95, sum(rate(http_server_duration_bucket{http_server_name!=\"rpc\"}[5m])) by (http_server_name, le))",
+          "expr": "histogram_quantile(0.95, sum(rate(http_server_duration_bucket{}[5m])) by (http_server_name, le))",
```

Comment:
@brunocalza I'm not sure about how this line and really all the changes to this file.  Is there a way to check if these changes are working how we want?